### PR TITLE
Write to allure environment file

### PR DESCRIPTION
**Pull request by elias-shoursoh**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by nirtal85 — CHANGES_REQUESTED** [review]
no allure env created 

see:
https://nirtal85.github.io/Selenium-Python-Example/110/#

File: tests/conftest.py [hunk 1]
```diff
@@ -41,11 +50,13 @@ def pages():
 
 
 @pytest.fixture(autouse=True)
+# Performs setup and tear down
 def create_driver(prep_properties, request):
     global driver
     browser = request.config.option.browser
     config_reader = prep_properties
     base_url = config_reader.config_section_dict("Base Url")["base_url"]
+    write_to_allure_env_file({"browser": browser, "base_url": base_url})
```
Comment (nirtal85) [inline tests/conftest.py:59]: we want it one time per session not for each test
use:
https://stackoverflow.com/a/32573090/4515129

File: globals/dir_global.py [hunk 2]
```diff
@@ -3,3 +3,7 @@
 ROOT_DIR = os.path.dirname(os.path.dirname(__file__))
 INI_CONFIGS_PATH = os.path.join(ROOT_DIR, "ini_configs")
 DATA_FILES_PATH = os.path.join(ROOT_DIR, "data")
+ENV_FILE_PATH = os.path.join(ROOT_DIR, "allure-results")
```
Comment (nirtal85) [inline globals/dir_global.py:6]: change to ALLURE_RESULTS_PATH

File: utils/config_parser.py [hunk 3]
```diff
@@ -20,3 +23,16 @@ def config_section_dict(self, section):
                 print(f"exception found in {key}")
                 section_dict[key] = None
         return section_dict
+
+
+class EnvironmentParser:
```
Comment (nirtal85) [inline utils/config_parser.py:28]: call this class AllureEnvironmentParser

**Review by nirtal85 — CHANGES_REQUESTED** [review]

File: tests/conftest.py [hunk 4]
```diff
@@ -22,13 +23,26 @@ def pytest_addoption(parser):
...
 @pytest.fixture(scope="session")
+# instantiates ini file parses object
 def prep_properties():
     config_reader = ConfigParserIni("props.ini")
     return config_reader
 
 
+@pytest.fixture(scope="session")
+# fetch browser kind, base url and writes a dictionary of key values into allure's environment.properties file
+def get_testing_properties(request, prep_properties):
```
Comment (nirtal85) [inline tests/conftest.py:34]: fixture name should be == write_allure_enviorment

**Review by nirtal85 — CHANGES_REQUESTED** [review]

File: tests/conftest.py (hunk 4 above)
Comment (elias-shoursoh) [reply tests/conftest.py]: done

**Review by nirtal85 — APPROVED** [review]

**Pull request merged**